src/App.css

```diff
@@ -1,45 +1,38 @@
-#root {
-  max-width: 1280px;
-  margin: 0 auto;
-  padding: 2rem;
-  text-align: center;
+@import url('https://fonts.googleapis.com/css2?family=Raleway:ital,wght@0,100..900;1,100..900&display=swap');
```

Comment:
Great for getting us off the ground, especially when much of the figma design needs to be scraped for font family's and values. When we get closer to production, we can optimize by moving to a cdn or self-hosting our font files.